### PR TITLE
Add page about myself

### PR DESCRIPTION
**Pull request by matthiasbeyer**
User page about myself.

@TheNeikos feel free to point me to typos ...

### CODE REVIEW

File: content/profiles/Matthias_Beyer.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - ruby
+  - tiling-windowmanager
+  - tmux
+  - vim
+---
+
+I entered the university in 2013, summer semester. I've been a member of the
+unfug from the very start of my first semester and I took the leadership of
+the unfug in my 4th semester. It was always a pleasure to me to hear talks
+with the people at the weekly meetups and although I had several topics for
```
Comment (TheNeikos) [inline content/profiles/Matthias_Beyer.md]: I think you meant 'heard' instead of 'had' no?

Comment (matthiasbeyer) [inline content/profiles/Matthias_Beyer.md]: Yup. That's exactly why someone should read it before merging.

Also, I'm not sure whether we should do this in German or english. What do you think?

Comment (TheNeikos) [inline content/profiles/Matthias_Beyer.md]: To be honest I think English is better, for example international students will be able to read it as well (which are here too short to need or actually be able to learn german)

Comment (matthiasbeyer) [inline content/profiles/Matthias_Beyer.md]: Good point. We shouldn't restrict members, anyways. I would say, english is fine if a member can provide her/his page in english, but if she/he only wants to write it in german, we go for it. Ack?

Comment (TheNeikos) [inline content/profiles/Matthias_Beyer.md]: Ack!

**Comment by matthiasbeyer** [comment]
@TheNeikos Any remaining notes? Or am I allowed to merge?

**Comment by TheNeikos** [comment]
You did not correct the mistake you acknowledged, once that is done I think so.

**Pull request merged**